.travis.yml

```diff
@@ -3,5 +3,8 @@ jdk:
   - openjdk8
 scala:
   - 2.12.10
-script: sbt clean coverage test coverageAggregate
+script:
+  - git tag 0.1234.5678
+  - sbt clean coverage test coverageAggregate
```

Comment:
What happens if this step fails for reason and the tag is never removed?

Reply:
tested and it works, I think because it runs build on new VM each time and does not push the tag